src/test/java/com/andreamazzarella/contact_manager/AgeTest.java

```diff
@@ -1,47 +1,55 @@
 package com.andreamazzarella.contact_manager;
 
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 public class AgeTest {
+    Age tenYears = new Age(10);
+    Age thirtyYears = new Age(30);
 
-    @Test
-    public void defaultsToZeroWithInvalidFormat() {
-        Age testAge = new Age("ciao");
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
```

Comment:
This is cool. I didn't know you could do this, I've used other, more verbose ways to test exceptions 👍